public/app/plugins/datasource/prometheus/components/monaco-query-field/MonacoQueryField.tsx

```diff
@@ -16,6 +16,7 @@ const options: CodeEditorMonacoOptions = {
   scrollBeyondLastLine: false,
   renderLineHighlight: 'none',
   fontSize: 14,
```

Comment:
What's the difference between `fontSize` and `suggestFontSize`? Do we need to set both?